overlay.d/05rhcos/usr/lib/dracut/modules.d/41rhcos-check-luks-syntax/rhcos-fail-boot-for-older-luks-syntax.service

```diff
@@ -0,0 +1,16 @@
+# Fixes the encryption issue with LUKS in 4.7 when a user 
+# tries to use the RHCOS 4.6 or earlier LUKS syntax in an 
+# Ignition config.
+[Unit]
+Description=Fail boot if Ignition uses an older syntax for enabling LUKS in 4.7 
+Before=ignition-disks.service
+After=ignition-fetch.service
+ConditionKernelCommandLine=ignition.firstboot
```

Comment:
The service is already activated only on firstboot via `ignition-complete.target`, so we don't really need this (doesn't hurt either of course, though might as well be consistent with other services like this).

Reply:
It makes sense to me!